packages/fe-mockserver-plugin-cds/README.md

```diff
@@ -48,6 +48,30 @@ You can also specify that you want this to generate a v2 service by using the fo
          mockDataPath: ./webapp/localService/mockdata
 ```
 
+## i18n
+
+If you use i18n reference in your project the plugin will automatically replace the i18n reference with the actual value from the i18n file. 
+The i18n file should be in the same folder as the metadata file and should be named `i18n.properties` 
+ - or in an i18n folder in the same directory as the metadata file.
+ - or in an _i18n folder in the same directory as the metadata file.
+ - or in an _i18n folder one level above the metadata file.
+ - or you can define i18nPath property in the configuration to add more location relative to the cds file
+
+```yaml
+ - name: sap-fe-mockserver
+     beforeMiddleware: compression
+     configuration:
+       metadataProcessor: 
+        name: "@sap-ux/fe-mockserver-plugin-cds"
+        options: 
+          odataVersion: v2
+       service:
+         urlPath: /here/goes/your/serviceurl
+         metadataPath: ./webapp/localService/metadata.cds
+         mockDataPath: ./webapp/localService/mockdata
+         i18nPath: ../otherlocation/i18n // relative to the metadata file
```

Comment:
The documentation does not match the implementation. Implementation looks for `i18nPath` in `options` not as part of the defined service. Did a little debugging session to find it 🧐 

```yaml
      configuration:
        debug: true
        metadataProcessor:
          name: "@sap-ux/fe-mockserver-plugin-cds"
          options:
            i18nPath: '../../srv/i18n' # <-- this is taken into account
        services:
          - urlPath: '/odata/v4/CapFeTsSampleService'
            metadataPath: './ui-annotations.cds'
            mockdataPath: './webapp/localService/mockdata'
            i18nPath: '../../i/am/being/ignored/sad/i18n' # <-- this is being ignored
```

I like the `i18nPath` as part of the service details more because then the i18n path could actually differ for each service in the array.

But at least the issue will be fixed with this change 🥳 

![image](https://github.com/user-attachments/assets/e42354b7-e302-4816-9fb7-e37563a4af4d)